presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -154,13 +154,11 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
     }
 
     @Override
-    protected SchemaTableName getSchemaTableName(ResultSet resultSet)
+    protected String getTableSchemaName(ResultSet resultSet)
```

Comment:
it would not be descriptive enough. I want to communicate this is "name of the schema of a table (from table list)", and not "name of a schema (from schema list)".